tests/unit/orcid/test_orcid_converter.py

```diff
@@ -139,7 +139,7 @@ def test_successfull_conversion():
                                                                        'external-identifier-type': 'ARXIV'}]},
                 'journal-title': 'Nuovo Cim.',
                 'short-description': 'Abstract',
-                'title': {'subtitle': 'Subtitle', 'title': 'Title3'},
+                'title': {'subtitle': 'Subtitle', 'title': 'Title'},
```

Comment:
Amending this because in the new definition, the main title is simply the first in 245